tests/normal_operation_prometheus.txt

```diff
@@ -23,9 +23,6 @@ http_requests_duration_seconds_count{endpoint="/_matrix/push/v1/notify",method="
 http_requests_total{endpoint="/_matrix/push/v1/notify",method="POST",status="200",otel_scope_name="axum-opentelemetry"} 8
 # TYPE notifications_total counter
 notifications_total{otel_scope_name="axum-opentelemetry"} 8
-# HELP otel_scope_info Instrumentation Scope metadata
-# TYPE otel_scope_info gauge
-otel_scope_info{otel_scope_name="axum-opentelemetry"} 1
 # HELP pushes_successful_total Successful pushes
```

Comment:
The commit message should ideally tell us why code was changed. What does this have to do with adding a CatchPanic handler?